### PR TITLE
Upgrade Ditto to 0.5.0

### PR DESCRIPTION
**Pull request by kerkhofsd**

### CODE REVIEW

File: alfresco-solrapi-client/alfresco-solrapi-client-ditto/src/integration-test/java/eu/xenit/alfresco/solrapi/client/ditto/DittoClientNoLiveNodeExistsReproduction.java [hunk 1]
```diff
@@ -58,9 +63,8 @@ public String createVersionableNode(String name) {
     @Override
     public void deleteNode(String liveNodeUuid) {
```
Comment (kerkhofsd) [inline alfresco-solrapi-client/alfresco-solrapi-client-ditto/src/integration-test/java/eu/xenit/alfresco/solrapi/client/ditto/DittoClientNoLiveNodeExistsReproduction.java:64]: AFAIK we don't have the functionality yet to implement this "in a clean way".

Feature request: `deleteNode` in Ditto.

Comment (tgeens) [reply alfresco-solrapi-client/alfresco-solrapi-client-ditto/src/integration-test/java/eu/xenit/alfresco/solrapi/client/ditto/DittoClientNoLiveNodeExistsReproduction.java:64]: ACK

Comment (driesdeproost) [reply alfresco-solrapi-client/alfresco-solrapi-client-ditto/src/integration-test/java/eu/xenit/alfresco/solrapi/client/ditto/DittoClientNoLiveNodeExistsReproduction.java:64]: can you make an issue in ditto referencing this PR?

**Review by driesdeproost — APPROVED** [review]

**Review by tgeens — APPROVED** [review]

File: alfresco-solrapi-client/alfresco-solrapi-client-ditto/src/integration-test/java/eu/xenit/alfresco/solrapi/client/ditto/DittoClientNoLiveNodeExistsReproduction.java (hunk 1 above)
Comment (kerkhofsd) [reply alfresco-solrapi-client/alfresco-solrapi-client-ditto/src/integration-test/java/eu/xenit/alfresco/solrapi/client/ditto/DittoClientNoLiveNodeExistsReproduction.java:64]: If you would like that :-)

Comment (driesdeproost) [reply alfresco-solrapi-client/alfresco-solrapi-client-ditto/src/integration-test/java/eu/xenit/alfresco/solrapi/client/ditto/DittoClientNoLiveNodeExistsReproduction.java:64]: Toon acking is fine as well :)

**Pull request merged**